pallets/thea-staking/src/lib.rs

```diff
@@ -288,6 +315,76 @@ pub mod pallet {
 			Self::deposit_event(Event::<T>::NetworkRemoved { network });
 			Ok(())
 		}
+
+		/// Allows active relayers report one another for free of any predefined misbehavior
+		/// If number of reporters is >= of given coeficient - slashing for pre-set coeficient
+		/// happens If reporter is part of active set - this call's fee is not apply
+		/// Full weight is payed on error
+		#[pallet::call_index(9)]
+		#[pallet::weight(1_000_000)]
+		pub fn report_offence(
+			origin: OriginFor<T>,
+			network_id: u8,
+			offender: T::AccountId,
+			offence: TheaMisbehavior,
```

Comment:
Please add params into the dockblock as it is in other methods.

Reply:
done